tests/hooks/pager.spec.ts

```diff
@@ -104,6 +150,48 @@ describe('Hook `usePager`', () => {
         checkPage({ callNumber: 1, pageNumber: 2, searchParams });
     });
 
+    test('method `loadPrevious`', async () => {
+        const data = success({
+            id: 'fakeID',
+            resourceType: 'type',
+            link: [
+                {
+                    relation: 'first',
+                    url: '/EpisodeOfCare?page=1',
+                },
+                {
+                    relation: 'next',
+                    url: '/EpisodeOfCare?page=3',
+                },
+                {
+                    relation: 'self',
+                    url: '/EpisodeOfCare?page=2',
+                },
+                {
+                    relation: 'previous',
+                    url: '/EpisodeOfCare?page=1',
+                },
+            ],
+        });
+
+        const searchParams = { a: 1, b: 2 };
```

Comment:
Seems like these search parameters are unused here and can probably be replaced by

```suggestion
        const searchParams = { _page: 2 };
```